packages/react-components/react-rating-preview/src/RatingItem.ts

```diff
@@ -0,0 +1 @@
+export * from './components/RatingItem/index';
```

Comment:
this is 100% lint error and would fail pipeline. looks like you have killed component generation generator early ?